package test_4;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

import com.mysql.jdbc.exceptions.jdbc4.MySQLDataException;

public class Driver {
	static String url = "jdbc:mysql://localhost:3306/test";
	static String user = "root";
	static String password = "541548";
	
	public static void main(String[] args) {
		try {
			Connection myConnection = DriverManager.getConnection(url, url, password);
			
			Statement myStatement = myConnection.createStatement();
			
			String sql = "delete from employees"
					+ " where first_name='Brown'";
			
			int rowsAffected = myStatement.executeUpdate(sql);
			
			System.out.println("Affected rows are: ");
		} catch (SQLException e) {
			e.printStackTrace();
		}
	}
	
}
